src/Resources/skeleton/test/Functional.tpl.php

```diff
@@ -10,16 +10,24 @@
 
 class <?= $class_name ?> extends <?= $panther_is_available ? 'PantherTestCase' : 'WebTestCase' ?><?= "\n" ?>
 {
-    public function testSomething()
+    public function testSomething(): void
```

Comment:
these `void` on test methods are pure visual noise...

Reply:
I know you don't like them but my goal here is pragmatic: without that PHPStorm displays shows a ugly notice and I don't like that freshly generated code triggers warnings in the most popular IDE.

Reply:
psalm will also complain without declaring a return type.. Seeing as we declare return types for the majority of the generated methods, tests should be no different to keep consistency...

Reply:
the rise of the machines :)